controllers/upgrade.js

```diff
@@ -0,0 +1,45 @@
+const PouchDB = require('pouchdb-core');
+PouchDB.plugin(require('pouchdb-adapter-http'));
+
+const buildDbUrl = 'https://staging.dev.medicmobile.org/_couch/builds';
+const buildDb = new PouchDB(buildDbUrl);
+const targetDbUrl = process.env.COUCH_URL;
+const targetDb = new PouchDB(targetDbUrl);
+
+module.exports = (version, username) => {
+  console.log('upgrade()', `Upgrading to ${version}…`);
+
+  console.log('upgrade()', 'Fetching newDdoc…');
+  return buildDb
+    .get(version, { attachments:true })
+    .then(newDdoc => console.log('upgrade()', 'Fetched newDdoc.') || newDdoc)
+
+    .then(newDdoc => console.log('upgrade()', 'Fetching oldDdoc…') || newDdoc)
+    .then(newDdoc =>
+      targetDb.get('_design/medic')
+        .then(oldDdoc => console.log('upgrade()', 'Fetched oldDdoc.') || oldDdoc)
+
+        .then(oldDdoc => {
+          newDdoc.app_settings = oldDdoc.app_settings;
```

Comment:
@garethbowen Defaults are dealt with in API already. See [config.js](https://github.com/medic/medic-api/blob/master/config.js#L59-L81). This runs initially and live on a listen.

If this logic isn't correct we should make it correct, or delete it, as it certainly gives the impression it fulfils the same function that your code above does.